pageserver/src/tenant/timeline.rs

```diff
@@ -827,6 +740,13 @@ impl Timeline {
                 self.compact_level0(layer_removal_cs.clone(), target_file_size, ctx)
                     .await?;
                 timer.stop_and_record();
+
+                if let Some(remote_client) = &self.remote_client {
+                    // should any new image layer been created, not uploading index_part will
+                    // result in a mismatch between remote_physical_size and layermap calculated
+                    // size, which will fail some tests, but should not be an issue otherwise.
+                    remote_client.schedule_index_upload_for_file_changes()?;
```

Comment:
Re: separate PR for comment change.

More recent version of https://github.com/neondatabase/neon/pull/4938/files#r1304833688

Reply:
Good to close this one.